app/grandchallenge/algorithms/tasks.py

```diff
@@ -139,10 +140,10 @@ def create_algorithm_jobs_for_session(
     algorithm_editors = [algorithm_image.algorithm.editors_group]
 
     # Send an email to the algorithm editors and creator on job failure
-    linked_task = send_failed_session_jobs_notifications.signature(
+    task_on_success = send_failed_session_jobs_notifications.signature(
```

Comment:
shouldn't this be task_on_failure?

Reply:
You would think so, but that is not how the `send_failed_session_jobs_notifications` task was implemented. This was a straight search and replace, linked tasks were only ever run when the chain was successful.